src/test/infra/lttng/test_lttng.py

```diff
@@ -506,40 +527,50 @@ def test_compare_init_event(self):
         assert init_events[33]['_timestamp'] == 600 and \
             init_events[33]['_name'] == 'ros2_caret:rmw_implementation'
-        assert init_events[34]['_timestamp'] == 600 and \
-            init_events[34]['_name'] == 'ros2_caret:construct_executor'
-        assert init_events[35]['_timestamp'] == 600 and \
-            init_events[35]['_name'] == 'ros2_caret:construct_static_executor'
-        assert init_events[36]['_timestamp'] == 600 and \
-            init_events[36]['_name'] == 'ros2_caret:add_callback_group'
-        assert init_events[37]['_timestamp'] == 600 and \
-            init_events[37]['_name'] == 'ros2_caret:add_callback_group_static_executor'
-        assert init_events[38]['_timestamp'] == 600 and \
-            init_events[38]['_name'] == 'ros2_caret:callback_group_add_timer'
-        assert init_events[39]['_timestamp'] == 400 and \
-            init_events[39]['_name'] == 'ros2_caret:callback_group_add_subscription'
-        assert init_events[40]['_timestamp'] == 600 and \
-            init_events[40]['_name'] == 'ros2_caret:callback_group_add_subscription'
-        assert init_events[41]['_timestamp'] == 400 and \
-            init_events[41]['_name'] == 'ros2_caret:callback_group_add_service'
-        assert init_events[42]['_timestamp'] == 600 and \
-            init_events[42]['_name'] == 'ros2_caret:callback_group_add_service'
-        assert init_events[43]['_timestamp'] == 400 and \
-            init_events[43]['_name'] == 'ros2_caret:callback_group_add_client'
-        assert init_events[44]['_timestamp'] == 600 and \
-            init_events[44]['_name'] == 'ros2_caret:callback_group_add_client'
-        assert init_events[45]['_timestamp'] == 600 and \
-            init_events[45]['_name'] == 'ros2_caret:rclcpp_construct_ring_buffer'
-        assert init_events[46]['_timestamp'] == 600 and \
-            init_events[46]['_name'] == 'ros2:rclcpp_construct_ring_buffer'
-        assert init_events[47]['_timestamp'] == 600 and \
-            init_events[47]['_name'] == 'ros2_caret:rclcpp_buffer_to_ipb'
-        assert init_events[48]['_timestamp'] == 600 and \
-            init_events[48]['_name'] == 'ros2:rclcpp_buffer_to_ipb'
-        assert init_events[49]['_timestamp'] == 600 and \
-            init_events[49]['_name'] == 'ros2_caret:rclcpp_ipb_to_subscription'
-        assert init_events[50]['_timestamp'] == 600 and \
-            init_events[50]['_name'] == 'ros2:rclcpp_ipb_to_subscription'
+
+        if ros_version == 'jazzy':
+            assert init_events[RESULT_ORDER_ADD_JAZZY+0]['_timestamp'] == 600 and \
+                init_events[RESULT_ORDER_ADD_JAZZY+0]['_name'] == \
+                    'ros2_caret:callback_group_to_executor_entity_collector'
+            assert init_events[RESULT_ORDER_ADD_JAZZY+1]['_timestamp'] == 600 and \
+                init_events[RESULT_ORDER_ADD_JAZZY+1]['_name'] == \
+                    'ros2_caret:executor_entity_collector_to_executor'
```

Comment:
This function tests an ordinal comparison of initialisation tracepoints.
Is there any need to handle only jazzy tracepoints separately from other tracepoints?

Reply:
Separated code for jazzy